packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -2673,22 +2678,12 @@ export class ContainerRuntime
 
 		try {
 			// See commit that added this assert for more details.
-			// These calls should be made for all but chunked ops:
-			// 1) this.pendingStateManager.processPendingLocalMessage() below
-			// 2) this.resetReconnectCount() below
+			// These subsequent calls should be made for all but chunked ops:
```

Comment:
This line alone doesn't make sense IMO.  I'd just delete it.  If you want to explain anything, I would just say somthing like "RemoteMessageProcessor would have already reconstituted Chunked Ops into the original op type"